Sources/SuperwallKit/Models/Product/Product.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  ProductType.swift
+//  Superwall
+//
+//  Created by Yusuf Tör on 01/03/2022.
+//
+
+import Foundation
+
+/// The type of product.
+public enum ProductType: String, Codable, Sendable {
```

Comment:
Do we need to have these `public`

Reply:
Yeah `Product` is exposed in `PaywallInfo`

Reply:
Could you make it an `Int` enum so its compatible with objc?

Reply:
Done